src/components/SubsectionNav/index.tsx

```diff
@@ -103,33 +104,34 @@ const SubsectionNav: React.FC<SubsectionNavProps> = ({
 
     const currentPath = window.location.pathname;
 
+    if (!isParentPage) {
+      return currentPage.split("/").pop() === url.split("/").pop();
+    }
     return (
       currentPath === url ||
       currentPath === `${url}/` ||
-      new RegExp(`${url}$`).test(currentPath) ||
-      (!isParentPage && currentPath.includes(url))
+      new RegExp(`${url}$`).test(currentPath)
     );
   };
 
+  const setTooltipStrategy = () => {
```

Comment:
Do we need to update tree items to have `truncateTreeItem` set to `true` so that this gets used? At the moment it looks like we don't get any tooltips because the tree items are wrapping by default